requre/online_replacing.py

```diff
@@ -485,12 +523,21 @@ def apply_decorator_to_all_methods(decorator, regexp_method_pattern="test.*"):
 
     ref: https://stackoverflow.com/a/6307868
 
+    Also triggers `cassette_setup`/`cassette_teardown` method before/after method execution
+    to be able to manipulate cassette in a method shared between all test cases.
+    (We do not have access to cassette from regular setUp/tearDown method.)
     """
 
     def decorate(cls):
         for attr in cls.__dict__:
             if callable(getattr(cls, attr)) and re.match(regexp_method_pattern, attr):
-                setattr(cls, attr, decorator(getattr(cls, attr)))
+                setattr(
```

Comment:
this is exactly an example where "pre-commit code reformatting" confuses code :-)

Reply:
I've added another call with the long name so I don't see a big problem in this specific case.

Reply:
nope, it is not big problem, just issue of this reformatting, and splitting to more lines, what causes mi  worse readability of these lines.